### PR TITLE
Add Remote Desktop Manager Free

### PR DESCRIPTION
**Pull request by Phyyl**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1474)

### CODE REVIEW

**Comment by MarNicGit** [comment]
@Phyyl Validation is working as intended albeit a bit bad in it's feedback. The manifest is currently located in `manifests/Devolutions/2020.1.20.0.yaml` but the correct directory would be `manifests/Devolutions/RemoteDesktopManagerFree/2020.1.20.0.yaml`.

File: manifests/Devolutions/2020.1.20.0.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Id: Devolutions.RemoteDesktopManagerFree
+Name: Remote Desktop Manager Free
```
Comment (Berkeley-Reject) [inline manifests/Devolutions/2020.1.20.0.yaml:2]:
```suggestion
Name: RemoteDesktopManagerFree
```
I think you should move the yaml file into the `manifests/Devolutions/RemoteDesktopManagerFree` folder and change the name field to `RemoteDesktopManagerFree` in order to pass the validation test.

**Review by xiaoyang-sde — CHANGES_REQUESTED** [review]

File: manifests/Devolutions/2020.1.20.0.yaml (hunk 1 above)
Comment (MarNicGit) [reply manifests/Devolutions/2020.1.20.0.yaml]: The name and publisher fields are allowed to be different from the package name, the validation only checks if the ID matches the folder structure.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**